src/style.css

```diff
@@ -1 +1,57 @@
 /* add styles here */
+
+.body {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Avoid using tag names for styling (except `html` and `body`). Instead, use classes to apply styles. This helps maintain the separation of concerns and makes the styles more reusable and easier to maintain.